web/cms/templates/layouts/header.php

```diff
@@ -16,31 +18,44 @@
             <?php include($_SERVER['DOCUMENT_ROOT'].'/cms/templates/includes/svg/logo.svg'); ?>
         </a>
         <?php
+
             if (perch_layout_var('page', true) == 'home') {
                 $monthDate = date("1-m-Y", strtotime("+1 month"));
                 PerchSystem::set_var('todaysDate', date('U'));
                 PerchSystem::set_var('nextMonth', date("F", strtotime("+1 month")));
                 PerchSystem::set_var('nextEvent', date('d-m-Y', strtotime('thursday', strtotime($monthDate))));
-                $filters = [
-                    [
-                      'filter'     => 'date',
-                      'match'      => 'gte',
-                      'value'      => date('Y-m-d')
-                    ],
-                    [
-                      'filter'     => 'eventDisplayDate',
-                      'match'      => 'lte',
-                      'value'      => date('Y-m-d H:i:s')
-                    ],
-                ];
 
-                perch_collection('Events', [
-                    'template'   => 'event_next.html',
-                    'sort'       => 'date',
-                    'sort-order' => 'ASC',
-                    'count'      => 1,
-                    'filter'     => $filters
+                $override_details = perch_content_custom('Event override', [
+                    'skip-template' => 'true',
+                    'return-html'   => 'true',
```

Comment:
Do you need to return html to get the data?

Reply:
good point. I'll remove it 👍